Assets/Modifiers/Bonus.csv

```diff
@@ -0,0 +1,13 @@
+Bonus Type,What it does,Modifier,Factor,Who,Obsolete
+Ph. D. Thesis in logarithmic time search,Index research shortened for all players,Days,-50,ALL,None
+Perl,All features speed up by 10%,*,0.10,SELF,None
+Java,All features speed up by 10%,*,0.10,SELF,None
+JSP,All features speed up by 10%,*,0.10,SELF,None
+.NET,All features speed up by 12%,*,0.12,SELF,JAVA
```

Comment:
Yeah I just didn't want the JAVA 10% faster and the .NET 12% faster modifier to stack up to 22%. Not sure how to do it better...